python/ray/_private/usage/usage_lib.py

```diff
@@ -28,29 +28,33 @@
 Usage data collection -> Various components (Ray agent, GCS, etc.) + usage_lib (cluster metadata).
 
 Usage report is currently "off by default". You can enable the report by setting an environment variable
-RAY_USAGE_STATS_ENABLE=1. For example, `RAY_USAGE_STATS_ENABLE=1 ray start --head`.
-Or `RAY_USAGE_STATS_ENABLE=1 python [drivers with ray.init()]`.
+RAY_USAGE_STATS_ENABLED=1. For example, `RAY_USAGE_STATS_ENABLED=1 ray start --head`.
+Or `RAY_USAGE_STATS_ENABLED=1 python [drivers with ray.init()]`.
 
-"Ray API server (currently a dashboard server)" reports the usage data to the hard-coded URL.
-`ray._private.usage.usage_constants.USAGE_REPORT_URL`.
+"Ray API server (currently a dashboard server)" reports the usage data to https://usage-stats.ray.io/.
 
-Data is reported
-- Every `ray._private.usage.usage_constants.USAGE_REPORT_INTERVAL_S`.
+Data is reported ever hour by default.
 
 Note that it is also possible to configure the interval using the environment variable,
-`RAY_USAGE_REPORT_INTERVAL_S`.
+`RAY_USAGE_STATS_REPORT_INTERVAL_S`.
 
 To see collected/reported data, see `usage_stats.json` inside a temp
```

Comment:
Is the json file ever updated? I see collect_timestamp_ms stays the same even with 1s reporting interval.

Reply:
It should be. Let me add tests.